### PR TITLE
sd-netlink: introduce sd_netlink_message_enter_array()

### PR DESCRIPTION
**Pull request by yuwata**
To read `CTRL_ATTR_MCAST_GROUPS` attribute. But the attribute is not read by any programs yet.

### CODE REVIEW

File: src/libsystemd/sd-netlink/netlink-message.c [hunk 1]
```diff
@@ -931,6 +924,36 @@ int sd_netlink_message_enter_container(sd_netlink_message *m, unsigned short typ
...
+        size_t size;
+        int r;
+
+        assert_return(m, -EINVAL);
+        assert_return(m->n_containers < RTNL_CONTAINER_DEPTH, -EINVAL);
+
+        r = netlink_message_read_internal(m, type_id, &container, NULL);
+        if (r < 0)
+                return r;
+        else
```
Comment (poettering) [inline src/libsystemd/sd-netlink/netlink-message.c]: this line be dropped... i mean you exit this function early here...

**Comment by yuwata** [comment]
@poettering Thank you for the review. Now, unnecessary 'else's are dropped.

**Comment by yuwata** [comment]
Hmm?? Force-push does not work...

**Comment by yuwata** [comment]
The 'minor suggestion' is resolved. Changing label.

**Pull request merged**